src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -184,7 +184,7 @@ class IOUAmountPage extends React.Component {
                         success
                         style={[styles.w100, styles.mt5]}
                         onPress={() => this.props.onStepComplete(this.state.amount)}
-                        isDisabled={this.state.amount.length === 0}
+                        isDisabled={this.state.amount < 0.01}
```

Comment:
So `this.state.amount` is actually assumed to be a String everywhere else in the code. Even though JS automatically casts it for you, I think it's still a good idea to cast it to a Float before doing this check. Also, the check was previously ensuring that the amount wasn't an empty string; even though your new check encompasses that (since NaN is always less than 0.01), I still think it's good to explicitly include it for clarity.
```suggestion
                        isDisabled={!this.state.amount.length || parseFloat(this.state.amount) < 0.01}
```